pkgs/by-name/he/heroic-epic-integration/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkgsCross,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "heroic-epic-integration";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "Etaash-mathamsetty";
+    repo = "heroic-epic-integration";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Zn0MsaQd8Ro6eu8IQkMcLNGLVTUukwajkn8PRLfB+Yw=";
```

Comment:
```suggestion
    tag = "v${finalAttrs.version}";
    hash = "sha256-pRgs1w4bzm5Ao0zXfaNxBAR8+h7w4I+C+bm4nT7kIgU=";
```

Reply:
Is there a difference between `rev` and `tag` on this fetcher?  I've seen a mix used throughout nixpkgs.

Reply:
Honestly don't know. I think it's preferred to use tag when available at this point.